docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -17,17 +17,14 @@
         wait_for_timeout: " {{ wait_for_timeout | default(400) }}"
         boot_wait_time: " {{ boot_wait_time | default(180) }}"
 
-    - set_fact:
```

Comment:
Suggest to keep this task, otherwise the later tasks may fail.

Further, expect to change the task in line102 and 103 to avoid using hard-coded values, instead, set the hard-coded path as the default values, so we can have them as overrides